frontend/src/scenes/surveys/SurveyCustomization.tsx

```diff
@@ -172,8 +172,8 @@ export function Customization({
                         value={appearance?.zIndex}
                         onChange={(zIndex) => onAppearanceChange({ ...appearance, zIndex })}
                         disabled={!surveysStylingAvailable}
-                        placeholder="99999"
-                        defaultValue="99999"
+                        placeholder="2147482647"
+                        defaultValue="2147482647"
```

Comment:
doing this change here to prevent issues that other customers had faced before, where the surveys is below other stuff